From 0d70a21c6f98575984c28f4e98a1fbf929195456 Mon Sep 17 00:00:00 2001
From: Jiri Kucera <jkucera@redhat.com>
Date: Thu, 25 Jan 2018 21:53:30 +0100
Subject: [PATCH] make: add $(DESTDIR) in installcheck target [bug #302]

This ensures the proper sox binary is invoked during tests.
---
 src/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 7cceaafd..caf98645 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -194,6 +194,6 @@ loc:
 # would run the test suite, but an uninstalled libltdl build cannot
 # currently load its formats and effects, so the checks would fail.
 installcheck:
-	$(srcdir)/tests.sh --bindir=${bindir} --builddir=${builddir} --srcdir=${srcdir}
-	$(srcdir)/testall.sh --bindir=${bindir} --srcdir=${srcdir}
+	$(srcdir)/tests.sh --bindir=$(DESTDIR)${bindir} --builddir=${builddir} --srcdir=${srcdir}
+	$(srcdir)/testall.sh --bindir=$(DESTDIR)${bindir} --srcdir=${srcdir}
 
-- 
2.22.0


